### PR TITLE
Fix brief stuttering when pressing Hotkey for the first time.

### PR DESCRIPTION
**Pull request by RedFlames**
Hello,

I've been experiencing a strange issue where if I have SpeedrunTool and JungleHelper loaded as mods, when I press a Speedrun Tool hotkey for the first time on the menu, the game briefly but noticable stutters. It doesn't happen when the two mods aren't in Zip-files and it seems to get worse with more and bigger mods loaded.

I've found a fix for it by making these FieldInfo reflection object get their value (if any) right away instead of having it happen through the Lazy<> class when the first button press checks for CelesteNetChatting.

Basically, the small stutter now most likely happens while mods are loaded before the splash screen ;)

Cheers,
RedFlames

### CODE REVIEW

**Comment by WEGFan** [comment]
I think using lazy here is because mods can be loaded in any order, so if Speedrun Tool is the first mod loaded, it can't get those `FieldInfo`s because the types aren't loaded into the assembly yet when constructing the class.

Maybe it's better to hook after a mod gets loaded and check if the mod is what we need, then initialize the `FieldInfo`s. There's a pull request in Everest to [add mod load and register events](https://github.com/EverestAPI/Everest/pull/408) when a mod gets loaded, but it's not merged yet so currently we can only use `On.Celeste.Mod.Everest.Loader.LoadMod`.

**Comment by DemoJameson** [comment]
thanks for the pr, but as WEGFan said there may be a problem, I will initialize these fields at the right time and not use lazy

**Comment by RedFlames** [comment]
Oh alright, that makes a lot of sense actually. But yes, the build that DemoJameson sent me on Discord, which I assume has this merged version of the fix in it, works fine for me. 👍 
Thanks for the quick responses!
